h2o-core/src/main/java/hex/Model.java

```diff
@@ -1430,7 +1484,7 @@ public double lift_top_group() {
    * @param catEncoded Whether the categorical columns of the test frame were already transformed via categorical_encoding
    */
   public static String[] adaptTestForTrain(final Frame test, final String[] origNames, final String[][] origDomains,
```

Comment:
@sebhrusen any inputs/concerns regarding this change?

Reply:
no, looks good! Actually it increases visibility on what this method is doing, limiting which parameters it's using.